libpius/signer.py

```diff
@@ -432,6 +433,9 @@ def encrypt_signed_uid(self, key, filename):
         # we get a ENC_INV.
         debug('Got GPG_KEY_EXP')
         continue
+      elif PiusSigner.GPG_KEY_CONSIDERED in line:
+        debug('Got KEY_CONSIDERED')
+        continue
```

Comment:
I think the KEY_CONSIDERED status code should also be handled, see https://git.gnupg.org/cgi-bin/gitweb.cgi?p=gnupg.git;a=blob;f=doc/DETAILS;h=a264e53eecf990cd2469a47bd95a2ff7bbfac1de;hb=c318561ef4c97f0c2767aef377531d58174060a1#l651
Just make sure the status code is zero.